acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/acceptance/dsl/condition/blockchain/ExpectBlockNotCreated.java

```diff
@@ -35,11 +31,13 @@ public ExpectBlockNotCreated(final EthTransactions eth) {
 
   @Override
   public void verify(final Node node) {
-    final BigInteger lastBlock = node.execute(eth.blockNumber());
-    Awaitility.await()
-        .ignoreExceptions()
-        .pollInterval(WAIT_TIME)
-        .atMost(30, TimeUnit.SECONDS)
-        .untilAsserted(() -> assertThat(node.execute(eth.blockNumber())).isEqualTo(lastBlock));
+    try {
+      Thread.sleep(INITIAL_WAIT);
+      final BigInteger lastBlock = node.execute(eth.blockNumber());
+      Thread.sleep(WAIT);
```

Comment:
This should be using a waiter that we expect to fail instead of a hard coded sleep - that way if a new block is published it fails immediately.  It's a small thing but not having to wait 15 seconds every time makes a big difference if you're trying to work out why the test is failing.

Reply:
Yeah not nice, but was simple. I've ended up changing it fail immediately if we get a new block.